### PR TITLE
OSDOCS-3319: add bug fix for BOOTIF parsing

### PR DESCRIPTION
**Pull request by jldohmann**
Version(s): 4.11

Issue: https://issues.redhat.com/browse/OSDOCS-3319

Link to docs preview: https://deploy-preview-44849--osdocs.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-11-release-notes#ocp-4-11-bug-fixes

Additional information: https://bugzilla.redhat.com/show_bug.cgi?id=2032717

### CODE REVIEW

**Review by dustymabe — APPROVED** [review]
LGTM

**Comment by jldohmann** [comment]
@miabbott could you PTAL for QE ack? Thank you!

**Review by miabbott — APPROVED** [review]
LGTM

**Comment by jldohmann** [comment]
0224a9f..789d5c3 grammatical tweaks from `vale`

File: release_notes/ocp-4-11-release-notes.adoc [hunk 1]
```diff
@@ -383,6 +383,8 @@ In the table, features are marked with the following statuses:
 [id="ocp-4-11-rhcos-bug-fixes"]
 ==== {op-system-first}
 
+* Before this update, if a machine was booted through PXE and the `BOOTIF` argument was on the kernel command line the machine would only boot with DHCP enabled on a single interface. With this update, the machine boots with DHCP enabled on all interfaces even if the `BOOTIF` argument is provided. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2032717[BZ#2032717])
```
Comment (kalexand-rh) [inline release_notes/ocp-4-11-release-notes.adoc:386]:
```suggestion
* Before this update, if a machine was booted through PXE and the `BOOTIF` argument was on the kernel command line, the machine would boot with DHCP enabled on only a single interface. With this update, the machine boots with DHCP enabled on all interfaces even if the `BOOTIF` argument is provided. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2032717[BZ#2032717])
```

**Review by kalexand-rh — COMMENTED** [review]
I have a small tweak, but this is looking good.

File: release_notes/ocp-4-11-release-notes.adoc (hunk 1 above)
Comment (jldohmann) [reply release_notes/ocp-4-11-release-notes.adoc]: Thanks Kathryn, fixed

**Pull request merged**